### PR TITLE
Disable deletion of "Output" data sources

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
Related to #1450

File: tomviz/PipelineView.cxx [hunk 1]
```diff
@@ -303,6 +303,14 @@ void PipelineView::contextMenuEvent(QContextMenuEvent* e)
...
   }
 
+  // Disable deletion for "Output" data sources
+  if (dataSource != nullptr) {
+    auto output = dataSource->property("output");
+    if (output.isValid() && output.toBool()) {
+      deleteAction->setEnabled(false);
+    }
+  }
+
```
Comment (mathturtle) [inline tomviz/PipelineView.cxx]: Shouldn't you just modify the enableDeleteItems() function?

Comment (cjh1) [reply tomviz/PipelineView.cxx]: Probably, thanks for the tip :smile_cat:

**Comment by cjh1** [comment]
@mathturtle That should be better

**Pull request merged**